Makefile

```diff
@@ -67,6 +67,16 @@ DEFS += -DNDEBUG
 DEFS += -DCONFIG_NFCT_PINS_AS_GPIOS
 DEFS += -DBUILD_VERSION='"$(BUILD_VERSION)"'
 DEFS += -DLFS2_NO_ASSERT
+DEFS += -DFIXED_POINT
```

Comment:
The floating point library might be working as well, and I think we have floating point hardware...